src/pp-annotator/viewer/AnnotationViewer.tsx

```diff
@@ -34,21 +49,21 @@ export default class AnnotationViewer extends Component {
                             onClick={(e) => this.props.callbacks.onDelete(e, this.props.annotation)}>Delete</button>
                 </span>
                 <div>
-                    {this.props.annotation.fields.annotationPriority}
+                    {(this.props.annotation.fields || {}).annotationPriority}
                 </div>
                 <div>
-                    {this.props.annotation.fields.comment}
+                    {(this.props.annotation.fields || {}).comment}
                 </div>
                 <div>
-                    {this.props.annotation.fields.link}
+                    {(this.props.annotation.fields || {}).link}
                 </div>
                 <div>
-                    {this.props.annotation.fields.linkTitle}
+                    {(this.props.annotation.fields || {}).linkTitle}
                 </div>
                 <div>
-                    {this.props.annotation.fields.isLinkOnly}
+                    {(this.props.annotation.fields || {}).isLinkOnly}
```

Comment:
cóż to za nieładne haczenie? :-)

jeśli już koniecznie potrzebne, może lepiej na start `render()` dać coś w stylu `const fields = this.props.annotation.fields || {};` ?

Reply:
to tymczasowy hack, już nie jest potrzebny bo zmieniłem interfejs